### PR TITLE
SEO Documentation page

### PR DESCRIPTION
**Pull request by khalilcodes**
## SEO Docs
Task from #57 
***

This PR adds documentation for SEO configuration in docs - [/docs/seo](https://deploy-preview-80--spectacular-dragon-c1015c.netlify.app/docs/seo)

### CODE REVIEW

**Review by rufuspollock — CHANGES_REQUESTED** [review]
The idea was to *just* have the documentation - not the rest of the stuff and then we add that once the docs are clear.

File: site/content/docs/seo.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: SEO Configuration
+description: How to set up SEO in flowershow
+---
+
+Flowershow uses the [NextSeo]( https://github.com/garmeeh/next-seo) plugin that helps to add seo properties to the page and is a fairly easy setup to follow.
```
Comment (rufuspollock) [inline site/content/docs/seo.md:6]: seo should be capitalized

File: site/content/docs/seo.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: SEO Configuration
+description: How to set up SEO in flowershow
+---
+
```
Comment (rufuspollock) [inline site/content/docs/seo.md:5]: Brief explanation of what we mean by SEO: e.g. 

Flowershow supports adding standard metadata to your pages to improve social media sharing and search engine discovery.

As well as setting values for individual pages you can add a default image and description that will be used on pages without this set.

File: site/content/docs/seo.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    facebook: {
+	   appId: '1234567890',
+    }
+  }
+}
+```
+
+More options can be found at https://github.com/garmeeh/next-seo#nextseo-options
+
+Note: It is advised to upload an image with the same width and height values as specified above for it to display properly.
```
Comment (rufuspollock) [inline site/content/docs/seo.md:39]: Add some info about what the result will be e.g. "When sharing on twitter you will get a result like this"

**Comment by khalilcodes** [comment]
I hadn't realized that the commits from the other branch were included here too. I will close this PR and open a clean one with the mentioned updates (as I think this will eventually lead to future merge conflicts).

**Pull request closed without merging**